web/templates/company/show.html.eex

```diff
@@ -18,6 +18,39 @@
   </li>
 
   <li>
+    <strong>Website:</strong>
+    <a href="<%= @company.website_url %>"><%= @company.website_url %></a>
+  </li>
+
+  <li>
+    <strong>Facebook:</strong>
+    <a href="http://www.facebook.com/<%= @company.facebook %>"><%= @company.facebook %></a>
+  </li>
+
+  <li>
+    <strong>Twitter:</strong>
+    <a href="http://www.twitter.com/<%= @company.twitter %>"><%= @company.twitter %></a>
+  </li>
+
+  <li>
+    <strong>Opening hours</strong>
+    <ol>
+      <%= if @company.opening_hours != nil do %>
```

Comment:
Can't we make sure this is never nil, but rather just an empty array if it doesn't exist? I feel like we have to keep checking for this nil all over the place...